wire-ios-notification-engine/WireNotificationEngineTests/BaseNotificationSessionTests.swift

```diff
@@ -148,7 +152,25 @@ class BaseTest: ZMTBaseTest {
             applicationStatusDirectory: applicationStatusDirectory,
             operationLoop: operationLoop,
             accountIdentifier: accountIdentifier,
-            pushNotificationStrategy: pushNotificationStrategy
+            pushNotificationStrategy: pushNotificationStrategy,
+            cryptoboxMigrationManager: mockCryptoboxMigrationManager
         )
     }
 }
+
+class MockCryptoboxMigrationManager: CryptoboxMigrationManagerInterface {
```

Comment:
move to own file?

Reply:
auto mocked this and moved to a new file